### DISCUSSION TITLE
Updating to Lit 2.0

### DISCUSSION THREAD

**Discussion by JimSpijker** [ing-bank/lion#1609] [Open for contributions]
Hi there, are there any near future plans of using Lit 2.0 instead of lit-element? Or are they already using Lit 2.0? Can't really seem to find it on the homepage.

**Comment by gerjanvangeest** [comment]
Hi Jim, we are already on Lit 2.0: https://github.com/ing-bank/lion/blob/master/packages/core/package.json#L37

But it can be that on some pages we still document the old ways. If you find any, please let us know, of a pull request would even be better!
